prototype/storage_api.py

```diff
@@ -0,0 +1,47 @@
+import mmif
+from fastapi import FastAPI
+from enum import Enum
+from pydantic import BaseModel
+from typing import List, Dict
+import os
+import json
+
+app = FastAPI()
+# get post request from user
+# read mmif inside post request, get view metadata
+# store in nested directory relating to view metadata
+
+# TODO: there's probably a better structure for this. Not sure if this works at the moment
+# TODO: but either way should hopefully be changed to utilize mmif operations?
+class Item(BaseModel):
+    views: List[Dict[str, any]]
+    documents: List[Dict[str, any]]
+    data: Dict[str, any]
+
+@app.get("/")
+async def root():
+    return {"message": "Storage api for pipelined mmif files"}
+
+@app.post("/upload_mmif/")
+async def upload_mmif(item:Item):
```

Comment:
this could accept a "single body" of MMIF json string and there should be a way, inside the function, we can deserialize the string into `Mmif` instance as usual.